Sources/IBLinterKit/Rules/UseBaseClassRule.swift

```diff
@@ -37,7 +37,7 @@ extension Rules {
             let violation: [Violation] = {
                 guard let baseClassesForElement = baseClasses[view.elementClass] else { return [] }
                 guard let customClass = view.customClass else {
-                    let message = "CustomClass is not set to \(viewName(of: view))"
+                    let message = "You must use a subclass of \(viewName(of: view))"
```

Comment:
I can move this to a separate PR if necessary. I found this warning message a little confusing because it says something like "CustomClass is not set to UILabel". Instead, we want to point out that they should change the class to be something other than UILabel.